enhancements/multi-arch/dynamic-imagestream-importmode-setting.md

```diff
@@ -0,0 +1,507 @@
+  change the `referencePolicy` to `Source`.
+- Scripts inspecting image objects and imagestream tags to
+  access metadata info like `dockerImageLayers` and
+  `dockerImageMetadata` will break and will need to change to
+  import arch specific tags to get metadata information.
+
+### Drawbacks
+
+- This change would mean users using the multi
+  payload would have their imagestream imports be
+  manifestlisted by default. For some users this
+  might be problematic if they are importing locally
+  and would have to deal with mirroring 4x the size
+  of the original single manifest image. In this
+  case, they might want to change the global to
+  `Legacy` and pick and choose which imagestreams
+  which would absolutely need to be manifestlisted.
+
+## Open Questions
+
+- How should CVO expose the payload type in the
+  Status section? is it through:
+    - string value of `single` and `multi` 
+    - individual architectures, i.e
+      `amd64`/`arm64`/`ppc64le`/`s390x` and
+      `multi`?
+    - array which lists all architectures for
+      `multi` and just one for single arch?
+      (in case there is ever a possibility of multiple
+      payloads with only certain architectures).
+
+## Test Plan
+
+1. e2e test in openshift-apiserver-operator for
+checking the observed config.
+2. e2e test in openshift-apiserver.
+3. QE test for CVO payload type status reporting
+on installs and upgrades.
+4. e2e test in the cluster-image-registry-operator
+for `imageStreamImportMode` setting in the cluster
+image config's status.
+4. QE test for installing a cluster on multi
+payload and creating imagestreams to check if the
+importMode is `PreserveOriginal` by default.
+5. QE test for single -> multi and multi -> single
+(not supported but can be explicit) upgrades and
+confirm appropriate imagestream modes for new
+imagestreams.
+
+## Graduation Criteria
+
+- Explicit documentation
+  (https://issues.redhat.com/browse/MULTIARCH-4560)
+  around this change is necessary to inform users on
+  the impact of installing/upgrading to a cluster
+  with multi payload.
+- implementation of e2e and QE test plans
+
+### Dev Preview -> Tech Preview
+
+N/A
+
+### Tech Preview -> GA
+
+N/A
+
+### Removing a deprecated feature
+
+N/A
+
+## Upgrade / Downgrade Strategy
+
+- For N->N+1 upgrades (assuming N+1 is the first
+  release of this feature), the new status field
+  will be available in the ClusterVersion
+
+## Version Skew Strategy
+
+- No issues expected with skews
+
+## Operational Aspects of API Extensions
+
+- The `imageStreamImportMode` API setting in the
+image config CRD's spec will not be documented. A
+cluster-fleet-evaluation[^2] would be done which accesses
+Telemetry data to determines the usage. If the field
+is not used widely after a set number of releases, deprecate
+and remove the field after appropriately informing the
+affected customers about the remediation.
+
+## Support Procedures
+
+### Cluster-fleet evaluation considerations
```

Comment:
@deads2k @wking added a stanza on the ClusterFleetEvaluation method and some details.

Reply:
cc @flavianmissi to take a look too as it involves the cluster-image-registry-operator reporting this condition.

Reply:
@flavianmissi it would be great if you could review this change to see if it looks ok to you.

Reply:
apologies for the delayed review - it lgtm!

Reply:
thanks!